src/wireviz/wireviz.py

```diff
@@ -17,7 +17,7 @@
 from wireviz.wv_helper import expand, open_file_read
 
 
-def parse(yaml_input: str, file_out: (str, Path) = None, return_types: (None, str, Tuple[str]) = None) -> Any:
+def parse(yaml_input: str, file_in: (str, Path) = None, file_out: (str, Path) = None, return_types: (None, str, Tuple[str]) = None) -> Any:
```

Comment:
Any reason why `Union[]` is preferred over `()` for type hints?
Especially since, AFAIK, they are not even enforced by Python, but are more of a documentation feature?

Reply:
Also, what's the meaning of `Tuple[str, ...]` towards the end of your suggestion?

Reply:
I know type hints are not enforced, but if we are going to use it in this project, I vote for using the same syntax where possible, and I have not seen any documentation of  `()` for type hints with alternative types, only `Union[]` - se links below:
- Background with PEP-links and video presentation by Guido van Rossum: [What are type hints in Python 3.5?](https://stackoverflow.com/questions/32557920/what-are-type-hints-in-python-3-5)
- Doc: [typing library doc](https://docs.python.org/3/library/typing.html)
- Quick overview: [Type hints cheat sheet (Python 3)](https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html)
```py
# For tuples of fixed size, we specify the types of all the elements
x: Tuple[int, str, float] = (3, "yes", 7.5)

# For tuples of variable size, we use one type and ellipsis
x: Tuple[int, ...] = (1, 2, 3)
```